stories/config.js

```diff
@@ -1,34 +1,44 @@
 // @flow
-
 //  Copyright (c) 2018-present, GM Cruise LLC
 //
 //  This source code is licensed under the Apache License, Version 2.0,
 //  found in the LICENSE file in the root directory of this source tree.
 //  You may not use this file except in compliance with the License.
 
-import { addDecorator, configure } from "@storybook/react";
-import { initScreenshot, setScreenshotOptions } from "storybook-chrome-screenshot";
+import { addDecorator, addParameters } from "@storybook/react";
+import React from "react";
+import { withScreenshot } from "storycap";
 
 import "webviz-core/src/styles/global.scss";
 import prepareForScreenshots from "./prepareForScreenshots";
 import storiesSetup from "webviz-core/src/stories/setup";
 import waitForFonts from "webviz-core/src/styles/waitForFonts";
 import installChartjs from "webviz-core/src/util/installChartjs";
 
+export const SCREENSHOT_VIEWPORT = {
+  width: 1001,
+  height: 745,
+};
+
 storiesSetup();
 global.GIT_INFO = {};
 installChartjs();
 
-addDecorator(initScreenshot());
-setScreenshotOptions({
-  // if this function is present on the page, we wait for the promise it returns to resolve before taking a screenshot.
-  waitFor: "waitFor",
-  delay: 100, // Small delay for rerenders that some components do.
-  viewport: {
-    width: 1001,
-    height: 745,
+addDecorator((storyFn) => {
+  // TODO: [EP-26051] This can be removed when modals remove themselves
```

Comment:
We usually avoid internal JIRA tags in the open source repo.

Reply:
Ah, good catch. I'll remove it here but leave it in the closed source repo 👍